release.json

```diff
@@ -1,14 +1,14 @@
 {
     "nightly": {
-        "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "master",
+        "INTEGRATIONS_CORE_VERSION": "7.20.x",
+        "OMNIBUS_SOFTWARE_VERSION": "7.20.x",
         "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
```

Comment:
Now that we correctly tag `omnibus-ruby`, the `omnibus-ruby` branch should also be switched to `7.20.x` (even if there's a very low chance that we actually commit anything new there), and we should also update the release docs to reflect that.

Reply:
It make sense. Will do.